pyfr/denserkschemes/__init__.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+from pkg_resources import resource_listdir, resource_string
+import re
+
+import numpy as np
+
+
+class BaseTabulatedDensePseudoStepper(object):
+    def __init__(self, rule):
+        a = []
+
+        rule = re.sub(r'(?<=\))\s*,?\s*(?!$)', r'\n', rule)
+        rule = re.sub(r'\(|\)|,', '', rule).strip()
+        rule = rule[1:-1] if rule.startswith('[') else rule
+
+        for l in rule.splitlines()[:-1]:
```

Comment:
Read all at once in a coeffs variable array and then split as a, b = coeffs[:-1], coeffs[-1]